sdk/nodejs/package.json

```diff
@@ -61,7 +59,21 @@
         "mocha": "^9.0.0",
         "nyc": "^15.1.0",
         "rome": "^12.1.0",
-        "sinon": "^14.0.0"
+        "sinon": "^14.0.0",
+        "ts-node": "^7.0.1",
+        "typescript": "~3.8.3"
```

Comment:
Make ts-node and typescript *devDependencies* so they are available while working on this repo.